<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebuggerWindow</class>
 <widget class="QDialog" name="DebuggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Debugger</string>
  </property>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stepButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runOneFrameButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Run One Frame</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runOneSecondButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Run One Second</string>
   </property>
  </widget>
  <widget class="QPushButton" name="breakpointsButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Breakpoints</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="memoryLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="memoryLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Memory</string>
   </property>
  </widget>
  <widget class="QLabel" name="registersLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Registers</string>
   </property>
  </widget>
  <widget class="QLabel" name="disassemblerLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Disassembler</string>
   </property>
  </widget>
  <widget class="QLabel" name="videoRegistersLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Registers</string>
   </property>
  </widget>
  <widget class="QLabel" name="otherRegisterslabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Other Registers</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="memoryWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>511</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="disassemblerWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>411</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="registersWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="videoRegistersWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>50</y>
     <width>111</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="otherRegistersWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>121</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
